### DISCUSSION TITLE
Paperless-ng in Proxmox Container ?

### DISCUSSION THREAD

**Discussion by mg1067** [jonaswinkler/paperless-ng#1303] [Support]
Hello,

I need help to install paperless in a proxmox container. I use a ubuntu 20.4 lxc container.
Have someone the right steps for me ?

Thanks

**Comment by TheFehr** [comment]
Did you try to use the ansible playbook? Never tried one with a LXC Container but probably should work: [docs about this](https://paperless-ng.readthedocs.io/en/latest/setup.html#setup-ansible)

Or then there is the "basic" bare metal install:  [docs about that](https://paperless-ng.readthedocs.io/en/latest/setup.html#bare-metal-route)

**Reply by mg1067** [reply]
Thanks..

I will test it..

**Reply by mg1067** [reply]
Hi... 
no chance. Is not running. I give up..
But thanks..
